plugins/CoreUpdater/Updater.php

```diff
@@ -168,6 +169,8 @@ public function oneClickUpdatePartTwo()
         try {
 
             if (Marketplace::isMarketplaceEnabled()) {
+                Updates_4_0_0_b1::ensureCorePluginsThatWereMovedToMarketplaceCanBeUpdated();
```

Comment:
when updating from 3.14.1 to 4.X then this method would be executed... we'll create the needed plugin.json files shortly before checking for updates so the plugins won't be marked as "bogus"